src/plugins/expressions/public/render.ts

```diff
@@ -92,6 +93,9 @@ export class ExpressionRenderHandler {
       event: (data) => {
         this.eventsSubject.next(data);
       },
+      getMode: () => {
```

Comment:
nit: I'd prefer consistency.

```suggestion
      getRenderMode: () => {
```